packages/flow-core/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 # Change Log
 
+## [1.5.4] - 2023-03-14
```

Comment:
Since from last week onwards we need to follow Semantic Versioning,
If features added then update minor version.
if improvements and bug fixes then update patch version
Since you added features change it to `1.6.0`